app/code/Magento/Catalog/Helper/Output.php

```diff
@@ -182,7 +182,7 @@ public function productAttribute($product, $attributeHtml, $attributeName)
         if ($attributeHtml !== null
             && $attribute->getIsHtmlAllowedOnFront()
             && $attribute->getIsWysiwygEnabled()
-            && $this->isDirectivesExists($attributeHtml)
+            && $this->isDirectivesExists((string)$attributeHtml)
```

Comment:
On the line 222 looks like we have the similar issue in `categoryAttribute` method.
https://github.com/magento/magento2/pull/27662/files#diff-a0d255192f82142f3e5cb7609365238dR222.

I think it's better to move casting to string inside method `isDirectivesExists`, so it will cover all cases at once

Reply:
I add casting to the string in categoryAttribute method. `isDirectivesExists` method should receive the string as a parameter. I understand that add casting inside `isDirectivesExists` is a complex fix but it's more looks like a hack. It isn't the best approach to have mixed type as parameter. I checked where this method used and it's just 2 entries in Magento source code